site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -87,50 +75,4 @@ You should see none (but no errors).
 To delete all resources provisioned by Terraform:
 ```
 terraform destroy
-```
-
-## Installing the Agones as Terraform submodule on Azure Kubernetes Service
-
-You can deploy Kubernetes cluster on Azure Kubernetes Service and install Agones using terraform.
-
-Install `az` utility by following [these instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
-
-The example of AKS submodule configuration could be found here:
- {{< ghlink href="examples/terraform-submodules/aks/module.tf" >}}Terraform configuration with Agones submodule{{< /ghlink >}}
-
-Copy `module.tf` file into a separate folder.
-
-Login to Azure CLI:
-```
-az login
-```
-
-Configure your terraform:
-```
-terraform init
-```
-
-Now you can deploy your cluster (use variables from the above `az ad sp create-for-rbac` command output):
-```
-terraform apply -var client_id="<appId>" -var client_secret="<password>"
-```
-
-Once you created all resources on AKS you can get the credentials so that you can use `kubectl` to configure your cluster:
-```
-az aks get-credentials --resource-group agonesRG --name test-cluster
-```
-
-Check that you have access to kubernetes cluster:
-```
-kubectl get nodes
-```
-
-### Uninstall the Agones and delete AKS cluster
-
-Run next command to delete all Terraform provisioned resources:
-```
-terraform destroy
-```
-
-### Reference
-Details on how you can authenticate your AKS terraform provider using official [instructions](https://www.terraform.io/docs/providers/azurerm/auth/service_principal_client_secret.html)
+```
```

Comment:
Please replace the "To verify agones was installed successfully" section (above, line 65 which I can't comment on) with a link to the new confirming page.